src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/Create/shipping.html.twig

```diff
@@ -56,13 +56,14 @@
         </span>
         </div>
         <div class="col-4">
-        <span class="ps-switch">
-          <input id="free-shipping_0" class="js-free-shipping-switch" name="free_shipping" value="0" type="radio">
-          <label for="free-shipping_0">{{ 'No'|trans({}, 'Admin.Global') }}</label>
-          <input id="free-shipping_1" class="js-free-shipping-switch" name="free_shipping" value="1" checked="" type="radio">
-          <label for="free-shipping_1">{{ 'Yes'|trans({}, 'Admin.Global') }}</label>
-          <span class="slide-button"></span>
-        </span>
+          <span class="ps-switch">
+            <input id="free-shipping_0" class="js-free-shipping-switch" name="free_shipping" value="0" type="radio">
+            <label for="free-shipping_0">{{ 'No'|trans({}, 'Admin.Global') }}</label>
+            <input id="free-shipping_1" class="js-free-shipping-switch" name="free_shipping" value="1" checked="" type="radio">
+            <label for="free-shipping_1">{{ 'Yes'|trans({}, 'Admin.Global') }}</label>
+            <span class="slide-button"></span>
+          </span>
+          <small class="js-free-shipping-switch-help form-text d-none">{{ 'Free shipping is already enabled by voucher'|trans({}, 'Admin.Shipping.Feature') }}</small>
```

Comment:
I cannot see where this text is displayed. Could you please make a screenshot? It might need harmonization, to match with the previous error notification, but I prefer having the screenshot first. :-)

Reply:
It displays when you have voucher added with free shipping enabled here:
![image](https://user-images.githubusercontent.com/32385991/75665805-6c8bf380-5c7d-11ea-9d8a-01e221d75317.png)

Reply:
Another possible case is when free shipping is toggled in the shipping section and you try to add the voucher with Free shipping, then different error message is displayed here: 
![image](https://user-images.githubusercontent.com/32385991/75666001-bbd22400-5c7d-11ea-830a-55ee2ffd37f8.png)

Reply:
Noted, thanks a lot, @tdavidsonas88! I prefer the first case, it looks more user-friendly. Let's pick this wording twice: _Free shipping voucher is already in your cart_ in _Admin.Orderscustomers.Help_ or _Admin.Notifications.Error_.

Reply:
Thank you Louise, changed it.